public/components/management/groups/multiple-agent-selector.tsx

```diff
@@ -498,6 +498,9 @@ export class MultipleAgentSelector extends Component {
                                 size='15'
                                 multiple
                                 onChange={(e) => {
+                                  document.getElementById("wzMultipleSelectorRight").childNodes.forEach(option =>  {
```

Comment:
Can you put all of this code in a new method and maybe you can reuse the method with param wzMultipleSelectorRight/wzMultipleSelectorLeft